src/Umbraco.Core/Manifest/ManifestContentAppDefinition.cs

```diff
@@ -112,6 +115,18 @@ public ContentApp GetContentAppFor(object o)
                 // else iterate over each entry
                 foreach (var rule in rules)
                 {
+                    if (rule.PartA == "role")
```

Comment:
can we make this an `InvariantEquals("role")`? rules are meant to be case-insensitive

Reply:
Good point! I have updated this 👍